inlong-tubemq/tubemq-client-twins/tubemq-client-go/client/consumer_impl.go

```diff
@@ -77,6 +77,10 @@ func NewConsumer(config *config.Config) (Consumer, error) {
 	if err := config.ValidateConsumer(); err != nil {
 		return nil, err
 	}
+	log.SetLogLevel(config.Log.LogLevel)
```

Comment:
What should we do if multiple consumers set Log information and the set values are inconsistent?

I personally think that the setting of the log service should be a separate operation and should not be bound to the Consumer object